db/dynamodb-global-table.yml

```diff
@@ -91,12 +97,22 @@ Parameters:
   TableName:
     Type: String
     Description: The unique DynamoDB table name.
-  PrimaryKey:
+  PartitionKey:
```

Comment:
The `PrimaryKey` parameter was a required parameter prior to this change, which means that if a stack deployed from this template were to be updated for some reason unrelated to this change, it would lose that parameter. At a minimum I would say that should be documented in the file, but to maintain compatibility it may be worth keeping the parameter name the same, and using the `ParameterLabels` and `Description` to denote that this value is actually a partition key, even if the name remains `PrimaryKey`.

Reply:
I moved everything for that param back to how it was, trying to make the most minimal change